src/Tasks/Microsoft.NET.Build.Tasks/ResourceAssemblyInfo.cs

```diff
@@ -13,5 +16,15 @@ public ResourceAssemblyInfo(string culture, string relativePath)
             Culture = culture;
             RelativePath = relativePath;
         }
+
+        public static ResourceAssemblyInfo CreateFromReferenceSatellitePath(ITaskItem referenceSatellitePath)
+        {
+            string destinationSubDirectory = referenceSatellitePath.GetMetadata("DestinationSubDirectory");
```

Comment:
What if the task item does not contain this metadata? Won't this produce a broken relative path?